pkg/daemon/daemon.go

```diff
@@ -333,6 +335,11 @@ func (d *Daemon) sharedErofsUmount(rafs *Rafs) error {
 		return errors.Wrapf(err, "umount erofs %s mountpoint, %s", err, mountpoint)
 	}
 
+	// delete fscache bootstrap cache file
+	if err := c.UnbindBlob("", fscaheID); err != nil {
+		log.L.Warnf("delete bootstrap %s err %s", fscaheID, err)
+	}
```

Comment:
Why ignoring the unbind error here?

Reply:
two reasons for this:
1. make  compatible with old nydusd.
2. Is this need to return err to containerd when delete bootstrap cache failed? seems we do not do this for blob cache

Reply:
Yes. It's a way to handle compatibility, however, it also hides the error if the nydusd supports this new API. Let's do it this way right now.